source/mainnet/smart-contracts/onboarding-guide-ethereum-developers/faq.rst

```diff
@@ -0,0 +1,371 @@
+        :width: 20 %
+    .. image:: ../tutorials/piggy-bank/images/pb_tutorial_6.png
+        :width: 20 %
+
+    **Option 2:** If you have the curl package installed on your Unix-like operating systems,
+    you can request CCD in the terminal directly from the wallet proxy via the below command.
+
+    .. code-block:: console
+
+        $curl -X PUT https://wallet-proxy.testnet.concordium.com/v0/testnetGTUDrop/<YourAccountAddress>
+
+    If you insert your account address correctly, the command should look similar to the below line.
+
+    .. code-block:: console
+
+        $curl -X PUT https://wallet-proxy.testnet.concordium.com/v0/testnetGTUDrop/4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JXBDAz9WdEy
+
+    **Option 3:**
+    If you have the curl package and the `concordium-client` tool installed on your Unix-like operating systems, you can request CCD to any of your alias account addresses.
+    If you already sent a previous request to the wallet proxy, you can not request any more CCD to the same account address.
+    Look up one of your alias account addresses instead and use it for your request.
+    The CCD will be credited to your canonical account address.
+
+    .. code-block:: console
+
+        $concordium-client account show-alias <YourAccountAddress> --alias <number>
+
+    If you insert your account address and a number correctly, the command should look similar to the below line.
+
+    .. code-block:: console
+
+        $concordium-client account show-alias 4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JaqovViXS4j --alias 17
+
+    This generates the output:
+
+    .. code-block:: console
+
+        The requested alias for address 4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JaqovViXS4j is 4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JXBDCpCaUT6
+
+    Copy your alias address to the below command.
+
+    .. code-block:: console
+
+        $curl -X PUT https://wallet-proxy.testnet.concordium.com/v0/testnetGTUDrop/<YourAliasAccountAddress>
+
+    If you insert your alias account address correctly, the command should look similar to the below line.
+
+    .. code-block:: console
+
+        $curl -X PUT https://wallet-proxy.testnet.concordium.com/v0/testnetGTUDrop/4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JXBDCpCaUT6
+
+    **Option 4:** If you need plenty of CCD for large-scale testing.
+    Please contact Concordium’s technical support via support@concordium.software.
+
+.. dropdown::  What does `invoke` mean?
+
+    `Invoke` may refer to:
+        - It can mean to execute or initiate a function. It is equivalent to Ethereum saying: "Calling a smart contract function".
+
+        - In the context of the `concordium-client` tool, it means to simulate a tx locally on your node via the `invoke` command of the `concordium-client` tool instead of sending the tx to the blockchain network and executing it on-chain. Since the tx was simulated it was not inserted by the bakers in a block and is not part of the blockchain.
```

Comment:
Perhaps make it even more explicit that any state changes that the invoke makes are discarded afterwards.